release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3349,6 +3349,71 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
+// 4.14.38
+=== RHSA-2024:7184 - {product-title} 4.14.38 bug fix and security update
+
+Issued: 3 October 2024
+
+{product-title} release 4.14.38, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:7184[RHSA-2024:7184] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:7187[RHSA-2024:7187] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.38 --pullspecs
+----
+
+[id="ocp-4-14-38-enhancements"]
+==== Enhancements
+
+The following enhancement is included in this z-stream release:
+
+[id="ocp-4-14-38-enhancements-_{context}"]
+===== Backporting configurable subnets
+
+* This release includes configurable subnets that allow you to configure the masquerade, as well as join and transition switch subnets to prevent overlaps with ip addresses that are used in your local infrastructure. (link:https://issues.redhat.com/browse/OCPBUGS-38440[*OCPBUGS-38440*])
```

Comment:
```suggestion
* This release includes configurable subnets that you can use to configure the masquerade. You can also use configurable subnets to join and transition subnets to prevent overlaps with IP addresses that are used in your local infrastructure. (link:https://issues.redhat.com/browse/OCPBUGS-38440[*OCPBUGS-38440*])
```

Reply:
"Allow" is a form of anthrophomorphism.

![Screenshot from 2024-10-03 17-15-40](https://github.com/user-attachments/assets/0cff9545-d39c-4689-b4bb-2121824f15d9)